app/helpers/news_helper.rb

```diff
@@ -18,4 +18,36 @@ def news_user(news)
       end
     end
   end
+
+  def news_button_text(category)
+    if category.present?
+      category
+    else
+      I18n.t('news.categories')
+    end
```

Comment:
Yeah, there is the same helper for documents. I am not sure how to make it generic, that's why I left it like this. 
If you have any idea how to make a better category dropdown I would need some help 😄